lib/perl/Genome/Model/Build/ClinSeq/InputBuilds.pm

```diff
@@ -30,4 +30,9 @@ sub resolve_rnaseq_builds {
   $rnaseq_builds->{$normal_build->id}{type} = 'rnaseq' if $normal_build;
 }
 
+sub wgs_exome_build {
```

Comment:
This name is a little strange.  The other `..._build`s return builds, but this returns a boolean.  I see how it's programmatically convenient in the rest of this commit, though.